docs/vectorizer/overview.md

```diff
@@ -344,6 +345,56 @@ CREATE TABLE blog_contents_embeddings_store(
 );
 ```
 
+## Destination Options for Embeddings
+
+Vectorizer supports two different ways to store your embeddings:
+
+### 1. Table Destination (Default)
+
+The default approach creates a separate table to store embeddings and a view that joins with the source table:
+
+```sql
+SELECT ai.create_vectorizer(
+    'blog'::regclass,
+    name => 'blog_vectorizer',  -- Optional custom name for easier reference
+    loading => ai.loading_column('contents'),
+    destination => ai.destination_table(
+        target_schema => 'public',
+        target_table => 'blog_embeddings_store',
+        view_name => 'blog_embeddings'
+    ),
+    embedding => ai.embedding_ollama('nomic-embed-text', 768)
+);
+```
+
+**When to use table destination:**
+- When you need multiple embeddings per row (chunking)
+- For large text fields that need to be split
+- You are vectorizing documents (which typically require chunking)
+
+### 2. Column Destination
+
+For simpler cases, you can add an embedding column directly to the source table:
```

Comment:
can you move some of the language I suggested above down here? Let's make it crystal-clear this is the no-chunking case